pom.xml

```
@@ -1,227 +1,239 @@
+                            <finalName>${project.artifactId}-${unqualifiedVersion}.${buildQualifier}</finalName>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.eclipse.tycho</groupId>
+                <artifactId>target-platform-configuration</artifactId>
+                <version>${tycho.version}</version>
+                <configuration>
+                    <target>
+                        <artifact>
+                            <groupId>org.corpus_tools</groupId>
+                            <artifactId>org.corpus_tools.hexatomic.target</artifactId>
+                            <version>0.2.0-SNAPSHOT</version>
+                        </artifact>
+                    </target>
+                    <environments>
+                        <environment>
+                            <os>linux</os>
+                            <ws>gtk</ws>
+                            <arch>x86_64</arch>
+                        </environment>
+                        <environment>
+                            <os>win32</os>
+                            <ws>win32</ws>
+                            <arch>x86_64</arch>
+                        </environment>
+                        <environment>
+                            <os>macosx</os>
+                            <ws>cocoa</ws>
+                            <arch>x86_64</arch>
+                        </environment>
+                    </environments>
+                </configuration>
+            </plugin>
+            <!-- USE GITFLOW -->
+            <plugin>
+                <groupId>com.amashchenko.maven.plugin</groupId>
+                <artifactId>gitflow-maven-plugin</artifactId>
+                <version>1.12.0</version>
+                <configuration>
+                    <tychoBuild>true</tychoBuild>
+                    <gpgSignTag>true</gpgSignTag>
+                    <gpgSignCommit>true</gpgSignCommit>
+                    <verbose>true</verbose>
+                    <skipFeatureVersion>true</skipFeatureVersion>
+                    <versionDigitToIncrement>1</versionDigitToIncrement>
+                    <pushRemote>true</pushRemote>
+                    <keepRemote>false</keepRemote>
+                    <digitsOnlyDevVersion>true</digitsOnlyDevVersion>
+                    <versionsForceUpdate>true</versionsForceUpdate>
+                    <gitFlowConfig>
+                        <versionTagPrefix>v</versionTagPrefix>
+                    </gitFlowConfig>
+                </configuration>
+            </plugin>
+            <!-- BUILD JAVADOCS -->
+            <plugin>
+                <!-- Parse version during `validate` phase to create the 
+                    properties - ${parsedVersion.majorVersion} - ${parsedVersion.minorVersion} 
+                    - ${parsedVersion.incrementalVersion} They are used during the aggregation 
```

```suggestion
                    - ${parsedVersion.incrementalVersion}. 
                    They are used during the aggregation 
```
Improve readability while keeping the style valid.